content/docs/learning-guide/tutorials/postfx/use-postfx-shape-weight-modifier.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Use PostFX Shape Weight Modifier to Modify Camera Exposure
+linktitle: Use PostFX Shape Weight Modifier
+description: Use PostFX Shape Weight Modifier to modify exposure control in Open 3D Engine (O3DE).
+toc: true
+---
+
+In this example, you will set up a spherical post-processing volume that controls the camera's exposure. When the camera moves in and out of the volume, its exposure changes.
+
+Post-processing volumes require two entities: one that defines the PostFX and one that defines a PostFX volume. This example uses the following entities:
+
+- **PostFX**: This entity contains an **Exposure Control** component and a **PostFX Layer** component that set the default amount of light that the camera receives from the environment.
+  
+- **PostFX Volume**: This entity contains a **PostFX Shape Weight Modifier** component and a **Sphere Shape** component that define a PostFX volume. It also contains an **Exposure Control** component and a **PostFX Layer** component that adjust the camera exposure when the camera moves through the PostFX Volume.
+
+To use post-processing volumes: 
+
+1. In **O3DE Editor**, create a default level.
+
+2. Create an entity for the PostFX. Right click in **Perspective** and select **Create entity**.
+  
+3. In the new PostFX entity, add the following components:
+    
+    - PostFX Exposure Control
+    
+    - PostFX Layer
+
+4. In the Exposure Control component, set the **Manual Compensation** property to `1.0`. This sets a default, bright exposure for the camera.
+
+
```

Comment:
super nitpick: this is the only step that has two lines after it instead of one

Reply:
In this example, you will set up a spherical post-processing volume that controls the camera's exposure. When the camera moves in and out of the volume, the exposure applied will change.

Setting up a post-processing volume and blending between looks can be managed with three entities: one that defines the default PostFX 'look', a second that defines the modified exposure based PostFX 'look', and a third that defines a PostFX volume which is used to control the blending between looks. This example uses the following entities:

Me speaking: basically, I think we should promote best practices ... to better understand the patterns, we should encourage the user to setup and manage the default look.  And then a second modified look.  This will help form a better foundation that will help scale to more complex understanding.